android/android_browser_tests.gni

```diff
@@ -56,12 +56,14 @@ android_test_exception_sources = [
   "//brave/browser/renderer_context_menu/brave_spelling_menu_observer_browsertest.cc",
   "//brave/browser/search_engines/search_engine_provider_service_browsertest.cc",
   "//brave/browser/search_engines/search_engine_tracker_browsertest.cc",
+  "//brave/browser/speedreader/speedreader_browsertest.cc",
```

Comment:
we will need to enable that test once we add UI and save to preferences on Android. Please create a follow up issue to do that.

Reply:
Created brave/brave-browser/issues/28308.